src/backend/base/langflow/components/agents/XMLAgent.py

```diff
@@ -12,43 +13,38 @@ class XMLAgentComponent(LCToolsAgentComponent):
     icon = "LangChain"
     beta = True
     name = "XMLAgent"
-
     inputs = LCToolsAgentComponent._base_inputs + [
         HandleInput(name="llm", display_name="Language Model", input_types=["LanguageModel"], required=True),
+        DataInput(name="chat_history", display_name="Chat History", is_list=True, advanced=True),
         MultilineInput(
             name="user_prompt",
             display_name="Prompt",
-            value="""
-You are a helpful assistant. Help the user answer any questions.
-
+            value="""You are a helpful assistant. Help the user answer any questions.
 You have access to the following tools:
-
 {tools}
-
 In order to use a tool, you can use <tool></tool> and <tool_input></tool_input> tags. You will then get back a response in the form <observation></observation>
-
 For example, if you have a tool called 'search' that could run a google search, in order to search for the weather in SF you would respond:
-
 <tool>search</tool><tool_input>weather in SF</tool_input>
-
 <observation>64 degrees</observation>
-
 When you are done, respond with a final answer between <final_answer></final_answer>. For example:
-
 <final_answer>The weather in SF is 64 degrees</final_answer>
-
 Begin!
-
 Question: {input}
-
 {agent_scratchpad}
```

Comment:
Changing the prompt shouldn't be necessary and is probably not a good idea.

Reply:
Actually, I just removed the first "enter" space because it was not good to see on the UI. It was just formatting changes.. In this case is It ok to format the prompt?

![image](https://github.com/user-attachments/assets/8d96a8c3-9d36-4a64-8c38-0bed8912427b)